core/shared/src/main/scala-3/zio/prelude/Liftables.scala

```diff
@@ -37,10 +37,10 @@ trait Liftables {
         case '{ Assertion.Regex.digit }                                                                => Some(Assertion.Regex.digit)
         case '{ Assertion.Regex.nonDigit }                                                             => Some(Assertion.Regex.nonDigit)
         case '{ Assertion.Regex.literal(${Expr(str)}) }                                                => Some(Assertion.Regex.literal(str))
-        case '{ Assertion.Regex.anyCharOf(${Expr(first)}, ${Expr(second)}, ${Varargs(rest)}: _*) }     => Some(Assertion.Regex.anyCharOf(first, second, Varargs(rest).valueOrError*))
-        case '{ Assertion.Regex.anyRegexOf(${Expr(first)}, ${Expr(second)}, ${Varargs(rest)}: _*) }    => Some(Assertion.Regex.anyRegexOf(first, second, Varargs(rest).valueOrError*))
-        case '{ Assertion.Regex.notAnyCharOf(${Expr(first)}, ${Expr(second)}, ${Varargs(rest)}: _*) }  => Some(Assertion.Regex.notAnyCharOf(first, second, Varargs(rest).valueOrError*))
-        case '{ Assertion.Regex.notAnyRegexOf(${Expr(first)}, ${Expr(second)}, ${Varargs(rest)}: _*) } => Some(Assertion.Regex.notAnyRegexOf(first, second, Varargs(rest).valueOrError*))
+        case '{ Assertion.Regex.anyCharOf(${Expr(first)}, ${Expr(second)}, ${Varargs(rest)}: _*) }     => Some(Assertion.Regex.anyCharOf(first, second, Varargs(rest).valueOrAbort*))
```

Comment:
Was generating the following scalac warning:
```scala
[warn] -- Deprecation Warning: /Users/jules/conduktor/workspace/zio-prelude/core/shared/src/main/scala-3/zio/prelude/Liftables.scala:40:152 
[warn] 40 |        case '{ Assertion.Regex.anyCharOf(${Expr(first)}, ${Expr(second)}, ${Varargs(rest)}: _*) }     => Some(Assertion.Regex.anyCharOf(first, second, Varargs(rest).valueOrError*))
[warn]    |                                                                                                                                                        ^
[warn]    |method valueOrError in trait Quotes is deprecated since 3.1.0: Use valueOrAbort
```